.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true
+}
```

Comment:
What happens if the extension isn't installed?

Reply:
I think nothing. the setting will be set but not used :)